### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by teifler**
Added contact information

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,16 +10,31 @@
...
+      repellendus vero, fugit voluptatibus.</figcaption>
+  </figure>
   <section id="contact">
     <h2>Contact</h2>
+    <ul>
+      <p>You want to be friends or contact me?</p>
+      <li>Email: <a href="mailto: tim.eif@protonmail.com"> tim.eif@protonmail.com</a></li>
+      <li><a href="https://ca.linkedin.com/in/tim-eifler">Linkedin</a></li>
+      <li><a href="https://github.com/teifler/tim-eifler">GitHub </a></li>
+    </ul>
```
Comment (mwarnck) [inline index.html]: I am not sure but I think the p element should not nested inside the ul element :)

File: index.html [hunk 2]
```diff
@@ -10,16 +10,31 @@
 <body>
   <header>
     <h1>Tim Eifler</h1>
+    <h3>Junior Webdeveloper</h3>
```
Comment (mwarnck) [inline index.html]: nice idea :)

**Review by lalueders — CHANGES_REQUESTED** [review]
Nice approach, I like! You use a h3 heading before a h2 heading. Would it be good to change the hierarchy of the headings?

**Review by MaggieSchuetz — COMMENTED** [review]
I agree with Mirco, otherwise it looks great!

File: index.html (hunk 1 above)
Comment (teifler) [reply index.html]: Absolutly right thanks for checking and noticing :)

**Comment by J-Massmann** [comment]
Your code looks good to me 😊  but I would agree with @lalueders, keep the hierarchy in your headings and otherwise style the `h2` with CSS if you don't like the standard style (even though we are probably gonna learn CSS not earlier than tomorrow 😂)

**Review by MaggieSchuetz — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -10,16 +10,44 @@
...
+    <figcaption>Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+      Explicabo voluptatem est nemo reprehenderit laborum libero quisquam 
+      iure facere, excepturi aperiam, ex totam eaque cumque maxime necessitatibus 
+      repellendus vero, fugit voluptatibus.</figcaption>
+  </figure>
   <section id="contact">
     <h2>Contact</h2>
+    <p>You want to be friends or contact me?</p>
+    <ul>
+      <li>Email: <a href="mailto: tim.eif@protonmail.com"> tim.eif@protonmail.com</a></li>
```
Comment (codingoak) [inline index.html:30]: the space after mailto: is not necessary but it works.

File: index.html [hunk 4]
```diff
@@ -10,16 +10,44 @@
...
+      <li><a href="https://ca.linkedin.com/in/tim-eifler">Linkedin</a></li>
+      <li><a href="https://github.com/teifler/tim-eifler">GitHub </a></li>
```
Comment (codingoak) [inline index.html:32]: ...and here after the link text GitHub

File: index.html [hunk 5]
```diff
@@ -10,16 +10,44 @@
 <body>
   <header>
     <h1>Tim Eifler</h1>
+    <h3>I'm a junior frontend Webdeveloper based in Würzburg, Germany</h3>
     <nav>
       <a href="#contact">Contact</a>
       <a href="#biographie">Biographie</a>
```
Comment (codingoak) [inline index.html:16]: biography instead of biographie (=german;) change it also in line 35, 36

File: index.html [hunk 6]
```diff
@@ -10,16 +10,44 @@
...
+    <ul>
+      <li></li>
+      <li></li>
+      <li></li>
+      <li></li>
+    </ul>
+  -->
   </section>
+  <footer>
+    <p>Tim Eifler - 2021 </p>
```
Comment (codingoak) [inline index.html:49]: space after 2021 not necessary

File: index.html [hunk 7]
```diff
@@ -10,16 +10,44 @@
...
+    <p><a href="mailto:tim.eif@protonmail.com"> tim.eif@protonmail.com</a></p>
```
Comment (codingoak) [inline index.html:50]: space before link text not necessary.

**Review by codingoak — COMMENTED** [review]
i would change the biographie to biography, rest is just nicer to look on the code. Nice work and good luck on the way to full stack dev ;)

**Pull request merged**